src/pages/trader-tools/common/_style.ts

```diff
@@ -573,7 +573,7 @@ export const FormulaContainerSwapMobile = styled(Flex)`
     }
 `
 
-export const FormulaHighlight = styled(Flex)`
+export const FormulaHighlight = styled(Flex)<{ gp?: string }>`
```

Comment:
what about `props.height`, `props.mt` and `props.pr` ?
i think we need to add them too

Reply:
There is no error for them but it's ok will add them as well

Reply:
Since we are using `Flex` here these props are in the `FlexType` so I guess we don't need to define them again, also there is no ts errors for them!